app/frontend/actions.js

```diff
@@ -233,14 +245,26 @@ module.exports = ({setState, getState}) => {
     const balance = await wallet.getBalance()
     const ownAddressesWithMeta = await wallet.getOwnAddressesWithMeta()
     const transactionHistory = await wallet.getHistory()
+    const conversionRates = getConversionRates(state)
 
     // timeout setting loading state, so that loading shows even if everything was cached
     setTimeout(() => setState({loading: false}), 500)
     setState({
       balance,
       ownAddressesWithMeta,
       transactionHistory,
+      conversionRates,
     })
+    try {
+      setState({
+        conversionRates: await conversionRates,
```

Comment:
you are doing an  await on an object, not on a promise. I suggest rewriting it like this:

`
let conversionRates = null
try {
    conversionRates = await getConversionRates(state)
} catch (e) {
    debugLog(`Could not fetch conversion rates:  ${e}`)
}
setState({
   balance,
   state,
   etc...
`